single-sign-on/index.md

```diff
@@ -33,6 +33,9 @@ Before you configure SSO for your organization, new members of your organization
 In addition, all email addresses should be added to your IdP.
 
 ## Configure
+The following video walks you through the process of configuring SSO.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/QY0j02ggf64" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 To configure SSO, log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} to obtain the **ACS URL** and **Entity IDs** to complete the IdP server configuration process. You can only configure SSO with a single IdP.  When this is complete, log back into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and complete the SSO enablement process.
```

Comment:
This section looks slightly hidden below the video frame. Wondering if we can move this below the section heading and then introduce the video. Just a thought.

Reply:
Thanks! @usha-mandya I rearranged the content with the video below.